### PR TITLE
feat: Style guide

### PR DESCRIPTION
**Pull request by lillianhidet**
#### Description

Addition of a style guide page to the wiki, which outlines best practices and style across the wiki to ensure consistent approaches across resources, as well as explaining how and when to add and use various features (Such as checklists, steps, etc)

### CODE REVIEW

File: src/content/docs/tuhura-tech/contribute.mdx [hunk 1]
```diff
@@ -62,6 +65,12 @@ At the start of the [number].mdx files
 type: tutorial
 title: 'My Page Title'
 ```
+To import necessary styles
```
Comment (Darkflame72) [inline src/content/docs/tuhura-tech/contribute.mdx:68]: Would these by styles or modules/components?

File: src/content/docs/tuhura-tech/style.mdx [hunk 2]
```diff
@@ -0,0 +1,342 @@
+---
+title: Contribution Best practices
```
Comment (Darkflame72) [inline src/content/docs/tuhura-tech/style.mdx:2]: Should be Contribution Style Best Practices

File: src/content/docs/tuhura-tech/style.mdx [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+---
+
+
+import Checklist from '/src/components/tutorial/Checklist.astro';
+import Box from '/src/components/tutorial/Box.astro';
+import { Steps } from '@astrojs/starlight/components';
+import { FileTree } from '@astrojs/starlight/components';
+
+:::note[Markdown]
+For more general info on Markdown syntax, take a look at the [Markdown Guide](https://www.markdownguide.org/basic-syntax/#overview)
```
Comment (Darkflame72) [inline src/content/docs/tuhura-tech/style.mdx:13]: Technically we allow a superset of markdown due to it being astro/starlight so we should link to that instead.

File: src/content/docs/tuhura-tech/style.mdx [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+
+There are two main types of contributions you'd be wanting to make:
+
+1. Single-page short overviews and guides
+2. Multi-page projects and tutorials
+
+Depending on what you're adding, some things will change, but we'll cover everything!
+
+:::note[Getting set up]
+For more information on how to get set up to contribute, take a look at the [Contribute doc](../contribute)
```
Comment (Darkflame72) [inline src/content/docs/tuhura-tech/style.mdx:29]: Shall we also link to the CONTRIBUTING.md on the github repo?

**Review by Darkflame72 — CHANGES_REQUESTED** [review]
Good base but a couple comments and we should link to the starlight doccs a bit more and make the actual "How to write eg diataxis more prominent".

File: src/content/docs/tuhura-tech/contribute.mdx (hunk 1 above)
Comment (lillianhidet) [reply src/content/docs/tuhura-tech/contribute.mdx:68]: Yeah think I was trying to oversimplify things, Modules works well.

**Review by Darkflame72 — APPROVED** [review]

**Pull request merged**